### PR TITLE
Consolidate WCF bindings

### LINKED ISSUE #9289: Consolidate WCF binding topic
## Consolidate WCF binding topics

The [\<binding> article](https://docs.microsoft.com/en-us/dotnet/framework/misc/binding) does not sync with the TOC and contains information more appropriate to the configuration file reference. It should be merged into the [\<bindings> article in the Configuration File Schema reference](https://docs.microsoft.com/en-us/dotnet/framework/configure-apps/file-schema/wcf/bindings), and requests for the article in the **misc** directory should redirect to the configuration file schema article. 

For the most part, the content of these two articles is identical. Any unique content in the article in the **misc** directory should be merged into the single article.

### PR DESCRIPTION
**Pull request by BillWagner**
Fixes #9289 

This PR resolves the merge conflicts in #10099. Those conflicts could not be merged in the GitHub UI.

### CODE REVIEW

**Review by rpetrusha — APPROVED** [review]
There's a problem with this topic, @BillWagner, which was why I left #10099 open. This single bindings topic is referenced from multiple locations (which was why it was in the misc folder), and it actually doesn't do a very good job of documenting the \<binding> element in a way that's consistent with the other config file reference documentation. 

Since this largely is #10099, I'll approve it, but I think it should probably be left open until we can decide how to address the issue of its multiple use and basic failure to document the \<binding> config element.

**Review by Lxiamail — APPROVED** [review]
I like the consolidation of the doc from misc to config-apps. Thanks for doing the fix. It looks good to me.

File: docs/framework/configure-apps/file-schema/wcf/bindings.md [hunk 1]
```diff
@@ -1,28 +1,32 @@
 ---
-title: "<bindings>"
-ms.date: "03/30/2017"
+title: "&lt;bindings&gt;"
```
Comment (mairaw) [inline docs/framework/configure-apps/file-schema/wcf/bindings.md]: This is an issue for Loc: https://github.com/dotnet/docs/pull/10137

Why do we need to roll back this change?

Comment (BillWagner) [reply docs/framework/configure-apps/file-schema/wcf/bindings.md]: When it was "<bindings>", the text was removed in the build process.

I'll try the "\<bindings\>"

Comment (mairaw) [reply docs/framework/configure-apps/file-schema/wcf/bindings.md]: It seems to be working fine as it is right now on docs: https://docs.microsoft.com/en-us/dotnet/framework/misc/binding

image

Comment (BillWagner) [reply docs/framework/configure-apps/file-schema/wcf/bindings.md]: I'll update to match that branch

**Comment by BillWagner** [comment]
@mairaw I think everything is updated correctly.

**Comment by mairaw** [comment]
Yep! Please go ahead and merge it.

**Pull request merged**